### ISSUE TITLE
antidea

### ISSUE THREAD

**Issue by wehro** [gregorio-project/hyphen-la#51]
*antid* in *antid-ea* is not a compound of *ante* and *id*, but an old form of *ante*. The hyphenation should be `an-tid-e-a`.

**Comment by fradec** [comment]
It seems to me that `an-ti-de-a` would be more correct, as is already the case. Do you mean that the `d` is an epenthesis?

**Comment by wehro** [comment]
No, `d` is no epenthesis here. *antid* is an archaic form of *ante* (according to *Stowasser*, 1998). `ea` is the well known demonstrative pronoun (*is/ea/id*).
`antidea` is a synonym of `antea`, which I propose to hyphenate `ant-e-a` instead of `an-te-a`, because it derives from a shortened *ante* and the pronoun *ea*.
